src/core/storage/storage-actions.ts

```diff
@@ -11,6 +11,8 @@ import {
 } from './repo'
 import { StorageWalletStatus } from './storage-reducer'
 
+export const syncInterval = 30 * 1000
```

Comment:
Shouldn't this be SYNC_INTERVAL by convention?

Reply:
Fixed.